src/main/java/org/minions/devfund/mauricioz/WhackAMole.java

```diff
@@ -0,0 +1,125 @@
+package org.minions.devfund.mauricioz;
+
+/**
+ * class in charge to provide game functionality.
+ */
+public class WhackAMole {
+    private static final int MOLES_LEFT = 10;
+    private int score = 0;
+    private int molesLeft;
+    private int attemptsLeft;
+    private char[][] moleGrid;
+
+    /**
+     * constructor.
+     * @param numAttempts number of attempts for the game.
+     * @param gridDimension grid size.
+     */
+    public WhackAMole(int numAttempts, int gridDimension) {
+        attemptsLeft = numAttempts;
+        moleGrid = new char[gridDimension][gridDimension];
+        this.molesLeft = MOLES_LEFT;
+        initializeMoleGrid();
+        placeMolesInTheGrid();
```

Comment:
the instructions say initialize the grid with the appropriate character
not say place the moles